diff --git a/node_modules/three/build/three.cjs b/node_modules/three/build/three.cjs
--- a/node_modules/three/build/three.cjs
+++ b/node_modules/three/build/three.cjs
@@ -19620 +19620 @@
-	if (typeof window !== 'undefined') animation.setContext(window);
+	if (typeof self !== 'undefined') animation.setContext(self);
diff --git a/node_modules/three/build/three.js b/node_modules/three/build/three.js
--- a/node_modules/three/build/three.js
+++ b/node_modules/three/build/three.js
@@ -19622 +19622 @@
-		if (typeof window !== 'undefined') animation.setContext(window);
+		if (typeof self !== 'undefined') animation.setContext(self);
diff --git a/node_modules/three/build/three.module.js b/node_modules/three/build/three.module.js
--- a/node_modules/three/build/three.module.js
+++ b/node_modules/three/build/three.module.js
@@ -26710 +26710 @@
-	if ( typeof window !== 'undefined' ) animation.setContext( window );
+	if ( typeof self !== 'undefined' ) animation.setContext( self );
diff --git a/node_modules/three/src/renderers/WebGLRenderer.js b/node_modules/three/src/renderers/WebGLRenderer.js
--- a/node_modules/three/src/renderers/WebGLRenderer.js
+++ b/node_modules/three/src/renderers/WebGLRenderer.js
@@ -883 +883 @@
-	if ( typeof window !== 'undefined' ) animation.setContext( window );
+	if ( typeof self !== 'undefined' ) animation.setContext( self );
